src/main/java/games/strategy/engine/data/properties/GameProperties.java

```diff
@@ -81,6 +84,9 @@ public Object get(final String key) {
     if (editableProperties.containsKey(key)) {
       return editableProperties.get(key).getValue();
     }
+    if (playerProperties.containsKey(key)) {
+      return playerProperties.get(key).getValue();
+    }
```

Comment:
Ah, my bad.  For some reason, I assumed that `constantProperties` were akin to defaults and `editableProperties` possibly overrode them.